packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -62,7 +62,7 @@ <h2 data-translate="manager_hub_dashboard_overview"></h2>
         </div>
         <h2 data-translate="manager_hub_dashboard_services"></h2>
         <hub-products
-            products="$ctrl.services.data"
+            products="$ctrl.products"
```

Comment:
data-

Reply:
This is a custom component so we've decided it is not mandatory